kubernetes/loculus/templates/lapis-silo-deployment.yaml

```diff
@@ -2,6 +2,9 @@
 {{- $keycloakTokenUrl := "http://loculus-keycloak-service:8083/realms/loculus/protocol/openid-connect/token" }}
 
 {{- range $key, $_ := (.Values.organisms | default .Values.defaultOrganisms) }}
+# TODO go through the schema for the organism and gather all the metadata entries that are of lineage type
+# TODO There can be multple fields, but they all need to have the same lineageSystem ... not sure how to handle yet
```

Comment:
As a starting point we could potentially just state that there can be only one `lineage` field

Reply:
your solution looks nice!

Reply:
I just added a commit, where I was thinking about having a new lineageSystems section in the Helm chart. It's then kind of easy to ensure only one system is sued per organism. Good, or a needless level of indirection?

Also happy to go with the "Only one lineage field" constraint for now (will make templating a bit simpler)

Reply:
Happy with all options. Until LAPIS/SILO supports multiple lineage systems, it's all a bit academic. We always can change it when LAPIS/SILO does implement something, so if it was me, I'd probably just have the simplest option for now of per-organism:

```
  name: organismName
  lineageDefs:
    1: http://....
```

but I don't mind